client/src/widgets/hooks/use-tc-send-function.tsx

```diff
@@ -0,0 +1,11 @@
+import { useEventBus } from '@wuespace/telestion-client-core';
+import { useCallback } from 'react';
+
+export function useTcSendFunction(channel: string) {
+	const eb = useEventBus(state => state.eventBus);
+
+	return useCallback(cmd => eb?.publish(channel, { content: cmd }), [
```

Comment:
The `className` property is missing.
The app requires it to decode the JSON string into a proper object.
I think, we need to talk with the application guys to resolve this issue. :wink: